packages/engineer/src/feature/dev-server.dev-server.env.ts

```diff
@@ -23,7 +23,8 @@ import { buildFeatureLinks } from '../feature-dependency-graph';
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const webpackDevMiddleware = require('webpack-dev-middleware') as (
-    compiler: webpack.MultiCompiler
+    compiler: webpack.MultiCompiler,
+    options?: { index?: string }
```

Comment:
is there any better type for this? this doesn't mean a lot like this.

maybe a more descriptive name, or type from 'webpack-dev-middleware' types

Reply:
I tried to bring in types from webpack-dev-middleware and webpack-hot-middleware but they have conflicts between those and webpack types